tests/test-env-py39.yml

```diff
@@ -46,7 +46,7 @@ dependencies:
   - pytest-httpserver
   - pytest-cov
   - pytest-timeout
-  - moto
+  - moto[server]
```

Comment:
not valid conda syntax, this only works with `pip`, unless something changed recently in conda

Reply:
Yeah, I had a feeling that was the case.  I've installed conda and I'll deal with it tomorrow.

Reply:
It's the moto version that's the issue - conda is pulling in an ancient version for some reason.

Reply:
Lol.  I just removed the ancient pin on `aiobotocore` and it built in 1/5 the time and works fine.